### DISCUSSION TITLE
How do I migrate from swiftmailer to symfony/mailer

### DISCUSSION THREAD

**Discussion by thelem** [symfony/symfony#45866] [Q&A]
I'm going through upgrading my symfony dependencies and I'm being told swift mailer is deprecated and I should use symfony mailer which sounds like it's just the next version with a new name. I can't find any migration guide though.

Is it just a matter of changing the package name? What other changes do I need to make?

**Comment by stof** [comment]
The API of symfony/mailer is different from the API of swiftmailer. Look at the doc of the mailer component to see how to use it.
